### PR TITLE
Home work 14

### PR DESCRIPTION
**Pull request by UnitOmega13**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import controller.ConsoleHandler;
+import controller.PlayerConsoleHandler;
+import dao.BetDao;
+import dao.BetDaoImpl;
+import dao.PlayerDao;
+import dao.PlayerDaoImpl;
+import javafx.util.Pair;
+import lib.Injector;
+
+import java.io.*;
```
Comment (Sofasmile) [inline src/Main.java]: try doesn't use autoimport

File: src/Main.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+public class Main {
+
+    static {
+        try {
+            Injector.injectDependency();
+        } catch (IllegalAccessException e) {
+            e.printStackTrace();
+        }
+    }
```
Comment (Sofasmile) [inline src/Main.java:14]: add a blank line after static block

File: src/dao/BetDao.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package dao;
+
```
Comment (Sofasmile) [inline src/dao/BetDao.java:2]: try to remove all redundant lines in this HW

File: src/dao/PlayerDao.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package dao;
+
+import model.Player;
+
+import java.util.List;
+
+public interface PlayerDao {
```
Comment (Sofasmile) [inline src/dao/PlayerDao.java:8]: optional: we can use one interface for two classes (BetDao, PlayerDao)

File: src/db/PlayerStorage.java [hunk 5]
```diff
@@ -0,0 +1,10 @@
+package db;
+
+import model.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlayerStorage {
+	public static final List<Player> playerList = new ArrayList<>();
```
Comment (Sofasmile) [inline src/db/PlayerStorage.java]:
```suggestion
	public static final List<Player> PLAYERS = new ArrayList<>();
```

File: src/db/Storage.java [hunk 6]
```diff
@@ -0,0 +1,11 @@
+package db;
+
+import model.Bet;
+
+import java.util.ArrayList;
+import java.util.List;
+
+//ЭТО НАША БД
+public class Storage {
+    public static final List<Bet> bets = new ArrayList<>();
```
Comment (Sofasmile) [inline src/db/Storage.java]:
```suggestion
    public static final List<Bet> BETS = new ArrayList<>();
```

Comment (UnitOmega13) [reply src/db/Storage.java]: Unfortunately, most classes wrote German, for the interface, he himself said to write it separately. 
Everything else is changed according to the comments.

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**